synapse/models/geospace.py

```diff
@@ -390,68 +390,64 @@ def repr(self, norm):
 
             ('geo:locatable', {
```

Comment:
Do we want to add a trilat property to account for Trilateration values? Wigle provides these for example, and it would allow us to more accurately reflect the data we get back vs storing them using latlong + accuracy

Reply:
This would definitely be an entirely new convention which is outside the scope of this PR. Feel free to create a discuss ticket.